src/app/shared/issue/description/description.component.ts

```diff
@@ -106,7 +106,9 @@ export class DescriptionComponent implements OnInit {
   private getUpdatedIssue(): Issue {
     return <Issue> {
       ...this.issue,
-      ['description']: this.issueDescriptionForm.get('description').value
+      ['description']: IssueService.getOrDefaultDescription(
+                                      this.issueDescriptionForm.get('description').value,
+                                      DEFAULT_BUG_REPORTER_DESCRIPTION)
```

Comment:
These seems more suited for an abstraction to be in the issue model. We can have an `updateDescription(updatedDescription)` method which acts as a black box. In the future, if there are more logic needed to clean the description, we just need to update it there. Same goes for the team response. This should remove the need for a static func and const in issue service.

What do you think?

Reply:
@JunWei96, thanks for the feedback!
It makes sense to me to place this logic in the issue model.
I'll go ahead and create the methods `updateDescription` and `updateTeamResponse` in the issue model.

Reply:
I've made the changes. Let me know if I misunderstood your meaning.
Thanks for your time!